### PR TITLE
Hub page

### PR DESCRIPTION
**Pull request by Mikejo5000**

### CODE REVIEW

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[c8bb141](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1537/commits/c8bb141941e10e4b6808656f6e1a40d30d8b686b)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/index.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-hub/docs/index.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/nuget/index?branch=pr-en-us-1537) | [Details](#user-content-d93b521a-46eb5d1a)

<a id="d93b521a-46eb5d1a"></a>
### [docs/index.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-hub/docs/index.md)
 - **Line 0**: **[Warning]** `````Invalid file link:(~/create-packages/creating-a-package-dotnet-cli.md).`````

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C7%5C15%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201907152130300356-1537%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=0W58pkM5JclaG%2BK4eAxXdFwVn5vj2jusXJ4Sv8OeBD0%3D&st=2019-07-15T21%3A26%3A22Z&se=2019-08-15T21%3A31%3A22Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[8341b14](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1537/commits/8341b1476d44d7bb0de5832c7f3d6997c30f94f1)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/index.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-hub/docs/index.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/nuget/index?branch=pr-en-us-1537) | [Details](#user-content-d93b521a-89e172ae)

<a id="d93b521a-89e172ae"></a>
### [docs/index.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-hub/docs/index.md)
 - **Line 0**: **[Warning]** `````Invalid file link:(~/host-packages/overview.md).`````
 - **Line 0**: **[Warning]** `````Invalid file link:(~/create-packages/creating-a-package-dotnet-cli.md).`````

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C7%5C15%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201907152138483234-1537%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=ISrgsPfVQ5nrPl64VYzhTXGguxDsraDrH3uQwA4q400%3D&st=2019-07-15T21%3A34%3A39Z&se=2019-08-15T21%3A39%3A39Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[2d234dd](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1537/commits/2d234ddf4cc944e87a454a0eec9998daa9726690)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/index.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-hub/docs/index.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/nuget/index?branch=pr-en-us-1537) | [Details](#user-content-d93b521a-56219378)

<a id="d93b521a-56219378"></a>
### [docs/index.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-hub/docs/index.md)
 - **Line 0**: **[Warning]** `````Invalid file link:(~/host-packages/overview.md).`````
 - **Line 0**: **[Warning]** `````Invalid file link:(~/create-packages/creating-a-package-dotnet-cli.md).`````

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C7%5C15%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201907152150161398-1537%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=lFsiKXbqctPO%2B9N7sl6FirvYuxcjYghKDQeqAtEEndE%3D&st=2019-07-15T21%3A46%3A11Z&se=2019-08-15T21%3A51%3A11Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[ec1faae](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1537/commits/ec1faae0430b6f2822702fcfb1d2feea6962280a)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/index.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-hub/docs/index.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/nuget/index?branch=pr-en-us-1537) | [Details](#user-content-d93b521a-e66462f5)

<a id="d93b521a-e66462f5"></a>
### [docs/index.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-hub/docs/index.md)
 - **Line 0**: **[Warning]** `````Invalid file link:(~/create-packages/creating-a-package-dotnet-cli.md).`````

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C7%5C15%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201907152155241769-1537%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=Syu9f65vsEgTiqsYu7DPFruYzDPFXa%2FG96sJlZV0tI4%3D&st=2019-07-15T21%3A51%3A16Z&se=2019-08-15T21%3A56%3A16Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[3b790b4](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1537/commits/3b790b465cd56afc3432b3adc4bb5ece9cca74c3)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/index.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-hub/docs/index.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/nuget/index?branch=pr-en-us-1537) | [Details](#user-content-d93b521a-4ed69cd8)

<a id="d93b521a-4ed69cd8"></a>
### [docs/index.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-hub/docs/index.md)
 - **Line 0**: **[Warning]** `````Invalid file link:(~/create-packages/creating-a-package-dotnet-cli.md).`````

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C7%5C15%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201907152203179287-1537%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=wNP0p6lZxhfb79cg0USTQgh0CVjXfTqvehVYC201Nx0%3D&st=2019-07-15T21%3A59%3A10Z&se=2019-08-15T22%3A04%3A10Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[64e73e2](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1537/commits/64e73e206599937a72a9dd6605381d91cb0ed074)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/index.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-hub/docs/index.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/nuget/index?branch=pr-en-us-1537) | [Details](#user-content-d93b521a-162e42f1)

<a id="d93b521a-162e42f1"></a>
### [docs/index.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-hub/docs/index.md)
 - **Line 0**: **[Warning]** `````Invalid file link:(~/refernence/nuget-exe-cli-reference.md).`````
 - **Line 0**: **[Warning]** `````Invalid file link:(~/create-packages/creating-a-package-dotnet-cli.md).`````
 - **Line 0**: **[Warning]** `````Invalid file link:(~/reference/api/overview.md).`````

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C7%5C16%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201907161628421976-1537%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=OoipsQpuCjUTPLVUSMtwCT8tD2kNPZkG5Xaxh207%2Flk%3D&st=2019-07-16T16%3A24%3A33Z&se=2019-08-16T16%3A29%3A33Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[e1a3487](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1537/commits/e1a34875a8724d8e71a59c919f6f0386e94ceb3a)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/index.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-hub/docs/index.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/nuget/index?branch=pr-en-us-1537) | [Details](#user-content-d93b521a-9b9ae6e0)

<a id="d93b521a-9b9ae6e0"></a>
### [docs/index.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-hub/docs/index.md)
 - **Line 0**: **[Warning]** `````Invalid file link:(~/refernence/nuget-exe-cli-reference.md).`````
 - **Line 0**: **[Warning]** `````Invalid file link:(~/create-packages/creating-a-package-dotnet-cli.md).`````
 - **Line 0**: **[Warning]** `````Invalid file link:(~/reference/api/overview.md).`````

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C7%5C16%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201907161631504519-1537%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=THe6CuMLQVW8f1DN%2FpdGDRbL0rniDOUllyJ0MVBh%2FMc%3D&st=2019-07-16T16%3A27%3A38Z&se=2019-08-16T16%3A32%3A38Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[208b5c2](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1537/commits/208b5c2ec3c5f86aa30c22aa8ce56dd239a85b1b)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/index.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-hub/docs/index.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/nuget/index?branch=pr-en-us-1537) | [Details](#user-content-d93b521a-de28357c)

<a id="d93b521a-de28357c"></a>
### [docs/index.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-hub/docs/index.md)
 - **Line 0**: **[Warning]** `````Invalid file link:(~/create-packages/creating-a-package-dotnet-cli.md).`````

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C7%5C16%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201907161634152012-1537%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=eXsm4C7JFwpABpAYlXkCc7qvExe%2B%2Bdc5UIBtDp%2FWGzM%3D&st=2019-07-16T16%3A30%3A09Z&se=2019-08-16T16%3A35%3A09Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[673fc6c](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1537/commits/673fc6c0471be8a3effbe938d8d032e0babb3c3c)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/index.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-hub/docs/index.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/nuget/index?branch=pr-en-us-1537) | [Details](#user-content-d93b521a-9a7c0a86)

<a id="d93b521a-9a7c0a86"></a>
### [docs/index.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-hub/docs/index.md)
 - **Line 0**: **[Warning]** `````Invalid file link:(~/create-packages/creating-a-package-dotnet-cli.md).`````

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C7%5C16%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201907161637459585-1537%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=3PKfu91qVdatDDVhcMjCmslzyhOOulUZiT%2BmVvW8ldw%3D&st=2019-07-16T16%3A33%3A41Z&se=2019-08-16T16%3A38%3A41Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[a23b620](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1537/commits/a23b62086a16daf5f2f25d5364338be87bfa2cb6)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/index.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-hub/docs/index.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/nuget/index?branch=pr-en-us-1537) | [Details](#user-content-d93b521a-1fc57337)

<a id="d93b521a-1fc57337"></a>
### [docs/index.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-hub/docs/index.md)
 - **Line 0**: **[Warning]** `````Invalid file link:(~/create-packages/creating-a-package-dotnet-cli.md).`````

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C7%5C16%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201907161650079148-1537%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=%2B67he1uHmq%2Fz%2FtadOFbZO0%2FjkAzQiQOmcQ2OOgDst9M%3D&st=2019-07-16T16%3A46%3A10Z&se=2019-08-16T16%3A51%3A10Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[50053e9](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1537/commits/50053e93db21b8fd59ecc727db6dbaa406a3823f)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/index.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-hub/docs/index.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/index?branch=pr-en-us-1537) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C7%5C23%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201907231822159341-1537%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=fdMc6LsRBhTGJObUZtdK7n65U4a5rcV2IeMMuCs4Z2A%3D&st=2019-07-23T18%3A18%3A12Z&se=2019-08-23T18%3A23%3A12Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/index.md [hunk 1]
```diff
@@ -5,240 +5,235 @@ author: karann-msft
...
 hide_bc: true
 ---
+
 <div id="main" class="v2">
-<div class="container">
-    <h1>NuGet Documentation</h1>
-    <ul class="pivots">
+    <div class="container">
+        <h1>NuGet Documentation</h1>
+        <p>NuGet allows developers to create, share, and consume useful .NET code. With NuGet, you can bundle code into "packages" that contain compiled code, such as DLLs.</p>
```
Comment (karann-msft) [inline docs/index.md]:
```suggestion
        <p>NuGet is the package manager for .NET. It enables developers to create, share, and consume useful .NET libraries. NuGet client tools provide the ability to produce and consume these libraries as "packages".</p> 
```

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[9b5d5fe](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1537/commits/9b5d5fe431526b48c4874a3dc3bb1c246b8f2bf0)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/index.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-hub/docs/index.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/index?branch=pr-en-us-1537) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C7%5C23%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201907232031318886-1537%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=WpirfgtURSzEnhuavEDO79hJ350luhKw3kKu8NjrNR8%3D&st=2019-07-23T20%3A27%3A33Z&se=2019-08-23T20%3A32%3A33Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[c0da7c9](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1537/commits/c0da7c95c7942a5f68325af6373a344970bf0163)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/index.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-hub/docs/index.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/nuget/index?branch=pr-en-us-1537) | [Details](#user-content-d93b521a-4d251e08)

<a id="d93b521a-4d251e08"></a>
### [docs/index.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-hub/docs/index.md)
 - **Line 0**: **[Warning]** `````Invalid file link:(~/nuget-org/data-requests.md).`````

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C7%5C24%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201907240024101826-1537%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=HNlslY2n9wLrHlnViNPnCPN%2BqkpFPJgMHvgnoQeQXUM%3D&st=2019-07-24T00%3A20%3A14Z&se=2019-08-24T00%3A25%3A14Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[ea8e768](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1537/commits/ea8e768de0730f8ba4e71c3d652d79695991c885)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/index.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-hub/docs/index.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/nuget/index?branch=pr-en-us-1537) | [Details](#user-content-d93b521a-92bdd85e)

<a id="d93b521a-92bdd85e"></a>
### [docs/index.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-hub/docs/index.md)
 - **Line 0**: **[Warning]** `````Invalid file link:(~/nuget-org/data-requests.md).`````

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C7%5C24%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201907240035428846-1537%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=B7wJwbEp7XxqPij4pcOTSOlXd3O9KDI5v80%2FIMQe7Yo%3D&st=2019-07-24T00%3A31%3A44Z&se=2019-08-24T00%3A36%3A44Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[21e14e6](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1537/commits/21e14e6d8615be25cfbe28c1e5a4e160fdecdaaf)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/index.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-hub/docs/index.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/index?branch=pr-en-us-1537) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C7%5C24%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201907240038504488-1537%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=uCvuoiKpkvl%2FUeVblIvTKN%2Bp%2FfHFDKAFwR17u%2Bjw3jk%3D&st=2019-07-24T00%3A34%3A46Z&se=2019-08-24T00%3A39%3A46Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[3b08728](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1537/commits/3b0872854ceba28237ef6743b483b24532637c65)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/index.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-hub/docs/index.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/index?branch=pr-en-us-1537) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C7%5C24%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201907240041536299-1537%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=A1PG5IdOv9nTmp8PGAbsTTZwtIUarnUShCXw%2BC%2FjX2E%3D&st=2019-07-24T00%3A37%3A47Z&se=2019-08-24T00%3A42%3A47Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[6f240d5](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1537/commits/6f240d5363eab6d34a3befb67c0e384986abd1ce)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/index.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-hub/docs/index.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/index?branch=pr-en-us-1537) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C7%5C24%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201907240046292668-1537%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=Lo8d5uMf5fSJaAUpZI8Gfr%2BA%2FHzisFKn3i0bpWDIcuU%3D&st=2019-07-24T00%3A42%3A26Z&se=2019-08-24T00%3A47%3A26Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by Mikejo5000** [comment]
@karann-msft Karan, please take a look. I believe this is ready to merge. I balanced the number of links a bit in each grouping, plus made all the requested updates.

File: docs/index.md [hunk 2]
```diff
@@ -5,240 +5,250 @@ author: karann-msft
...
+                    <div class="cardText">
+                        <h3><a href="policies/governance.md">Resources</a></h3>
+                        <p>
+                            <a href="policies/governance.md">Policies - NuGet</a>
+                        </p>
+                        <p>
+                            <a href="nuget-org/policies/data-requests.md">Policies - NuGet.org</a>
+                        </p>
+                        <p>
+                            <a href="release-notes/known-issues.md">Release notes</a>
```
Comment (karann-msft) [inline docs/index.md]:
```suggestion
                            <a href="release-notes/">Release notes</a>
```

Comment (karann-msft) [reply docs/index.md]: let's point the reader to list of release notes. The known issues page is quite out of date, I wouldn't want to update this link for every new release.

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[2071832](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1537/commits/20718327bff3f4d8cc043bf3631aa42dde2b6ed6)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/index.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-hub/docs/index.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/nuget/index?branch=pr-en-us-1537) | [Details](#user-content-d93b521a-cbedd324)

<a id="d93b521a-cbedd324"></a>
### [docs/index.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-hub/docs/index.md)
 - **Line 0**: **[Warning]** `````Invalid file link:(~/release-notes/).`````

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C7%5C24%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201907240332598002-1537%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=JOMnJRVNh2qI05Rasp55k4lFyWcXEfYV2S3EdRgVryY%3D&st=2019-07-24T03%3A29%3A01Z&se=2019-08-24T03%3A34%3A01Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**